backend/pyproject.toml

```diff
@@ -27,6 +27,7 @@ dev = [
     "httpx>=0.28.1",
     "prek>=0.2.10",
     "pytest>=8.4.2",
+    "pytest-asyncio>=1.2.0",
```

Comment:
Might not be required, but not bad to have just to be more explicit about how we're doing async **pytest**.